packages/archivematica_triggerer/.sqlfluff

```diff
@@ -0,0 +1,10 @@
+[sqlfluff]
```

Comment:
Do we define this elsewhere? If so, would it make sense for us to have common sqlfluff configs?

Reply:
Yes, good idea